cpp/libjsonnet++.cc

```diff
@@ -0,0 +1,157 @@
+// Copyright 2015 Google Inc. All rights reserved.
```

Comment:
I see what you mean. Makes sense. Also, if we end up refactoring libjsonnet.cpp and exposing both a C and C++ APIs, then that will essentially be the case.